src/d3a/models/strategy/__init__.py

```diff
@@ -451,9 +454,14 @@ def __init__(self):
         self._allowed_disable_events = [AreaEvent.ACTIVATE, MarketEvent.OFFER_TRADED]
         self.event_responses = []
         self._market_adapter = market_strategy_connection_adapter_factory()
+        self._settlement_market_strategy = self._create_settlement_market_strategy()
 
     parameters = None
 
+    @classmethod
+    def _create_settlement_market_strategy(cls):
+        return SettlementMarketStrategyInterface()
```

Comment:
What would happen if we call `settlement_market_strategy_factory` here as well? Wouldn't it be working the same way?